webapp/src/types/types.ts

```diff
@@ -180,3 +180,8 @@ export type RealNewPostMessageProps = {
     set_online: boolean;
     team_id: string;
 }
+
+export type StopJobData = {
+    channelID: string;
+    jobID: string;
+};
```

Comment:
Remind me why we're duplicating the type here?

Reply:
No reason, that should have been removed. I initially implemented it on this side only but then moved it to `calls-common` since it's a public client event and also removed the redundant channel ID which is not necessary.